### PR TITLE
Added 2 leetcode Solutions (206 and 208)

### PR DESCRIPTION
**Pull request by SadafKausar2025**
## Related Issue #1137

[Cite any related issue(s) this pull request addresses. If none, simply state "None”]

## Description

[Please include a brief description of the changes or features added]

## Type of PR

- [x Documentation update

## Screenshots / Videos (if applicable)

Screenshot_13-6-2024_18451_localhost

Screenshot_13-6-2024_18514_localhost

[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have followed the code style guidelines of this project.
- [x] I have checked for any existing open issues that my pull request may address.
- [x] I have ensured that my changes do not break any existing functionality.
- [x] Each contributor is allowed to create a maximum of 4 issues per day. This helps us manage and address issues efficiently.
- [x] I have read the resources for guidance listed below.
- [x] I have followed security best practices in my code changes.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Please read the following resources before submitting your contribution:

- [ ] [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [ ] [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [ ] [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [ ] [Markdown Guide](https://www.markdownguide.org/)
- [ ] [MDX Documentation](https://mdxjs.com/docs/)
- [ ] [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- [ ] [Tabs: Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [ ] [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [ ] [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex) 

## Contribution Guidelines

Thank you for considering contributing to our project! To ensure smooth collaboration and effective contribution management, please adhere to the following guidelines:

### Issue Creation

1. **Limit on Issues:**
   - Each contributor is allowed to create a maximum of **4 issues per day**. This helps us manage and address issues efficiently.

### Contribution Levels

2. **Basic Contributions:**
   - This project is primarily focused on documentation. Most of the setup has been completed, so contributors will generally need to work on basic code tasks, such as writing tests.
   - For these tasks, issues will be assigned the **level1** label.

3. **Acknowledging Hard Work:**
   - If a contributor puts in significant effort on a task, the issue will be upgraded to **level2**. This is our way of recognizing and appreciating extra effort.

4. **Feature Additions and Component Work:**
   - Contributors working on new features or components using JSX/TSX will be assigned a level based on the complexity and quality of their work.
   - The more complex and valuable the contribution, the higher the level assigned.

### Level Definitions

- **Level1:**
  - Tasks are straightforward, such as fixing minor bugs, writing tests, or making simple documentation updates.
- **Level2:**
  - Tasks require more effort, such as addressing complex bugs, improving existing features, or making substantial documentation improvements.
- **Level3:**
  - Tasks are highly complex and involve significant new feature development, major refactoring, or extensive contributions to the project’s core components.

We look forward to your contributions and appreciate your effort in helping us improve the project!

### CODE REVIEW

**Comment by SadafKausar2025** [comment]
hello @Ajay-Dhangar 
can you please review this PR and Merge it

File: dsa-solutions/lc-solutions/0200-0299/0206-Reverse-Linkedlist.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+          return 0;
+      }
+      ```
+    </TabItem>
+
+</Tabs>
+
+## Video lecture
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/D2vI2DNJGd8?si=fCV_x4Mwv--u4yUb" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/0200-0299/0206-Reverse-Linkedlist.md:299]: ### Bad Practice

```md
<iframe width="560" height="315" src="https://www.youtube.com/embed/D2vI2DNJGd8?si=fCV_x4Mwv--u4yUb" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```
### Good Practice

```md
<LiteYouTubeEmbed
    id="VIDEO_ID_HERE"
    params="autoplay=1&autohide=1&showinfo=0&rel=0"
    title="VIDEO_TITLE_HERE"
    poster="maxresdefault"
    webp />

    <br />
```

Comment (Ajay-Dhangar) [reply dsa-solutions/lc-solutions/0200-0299/0206-Reverse-Linkedlist.md:299]: for Example

```jsx
<LiteYouTubeEmbed
    id="lI3RBnK8V6Y"
    params="autoplay=1&autohide=1&showinfo=0&rel=0"
    title="CodeHarborHub Website Introduction | CodeHarborHub Website Overview | CodeHarborHub Website Features"
    poster="maxresdefault"
    webp />
```

and Output is

https://codeharborhub.github.io/docs/#what-is-codeharborhub

File: dsa-solutions/lc-solutions/0200-0299/0208-Implement-Trie.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+tags:
+  - Trie
+  - java
+  - python
+  - c++
+  - Recursion
+description: "This document provides a solution to the Implementation of Trie."
+---
+
+# 0208- Implement Trie (Prefix Tree)
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/0200-0299/0208-Implement-Trie.md:14]: remove `# 0208- Implement Trie (Prefix Tree)` line

File: dsa-solutions/lc-solutions/0200-0299/0208-Implement-Trie.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+          return 0;
+      }
+      ```
+    </TabItem>
+
+</Tabs>
+
+## Video Lecture:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/YXefcFMpemo?si=OKoeQHKTtnsy1yby" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/0200-0299/0208-Implement-Trie.md:303]: also, change for best and good practice like

```md
<LiteYouTubeEmbed
    id="VIDEO_ID_HERE"
    params="autoplay=1&autohide=1&showinfo=0&rel=0"
    title="VIDEO_TITLE_HERE"
    poster="maxresdefault"
    webp />

    <br />
```

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

File: dsa-solutions/lc-solutions/0200-0299/0206-Reverse-Linkedlist.md (hunk 1 above)
Comment (SadafKausar2025) [reply dsa-solutions/lc-solutions/0200-0299/0206-Reverse-Linkedlist.md:299]: what i will write in id params and poster

Comment (SadafKausar2025) [reply dsa-solutions/lc-solutions/0200-0299/0206-Reverse-Linkedlist.md:299]: oh I understood

**Comment by SadafKausar2025** [comment]
hello @Ajay-Dhangar 
please see I have corrected the code
Thank you for giving new learning,I learned a lot by doing this open-source program

**Comment by Ajay-Dhangar** [comment]
> hello @Ajay-Dhangar please see I have corrected the code Thank you for giving new learning,I learned a lot by doing this open-source program

Great

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**